corehq/apps/groups/templates/groups/all_groups.html

```diff
@@ -35,6 +35,16 @@
 {% endblock %}
 
 {% block main_column %}
+{% if needs_to_downgrade_locations %}
+    <div>
+        <p>
+            {% trans "This project no longer has access to the Organization feature." %}
+            <a href="{% url "downgrade_locations" domain %}">
+                {% trans "Click here to fix this." %}
```

Comment:
I want to make sure people really know what they are doing rather than just
click a "fixit" button. Making them nav to the Locations page I felt would
force them to confront what locations are and what they are undoing. I'm
imagining someone sees this who hasn't ever been on the Locations page
before.

That said, don't feel strongly so if you two agree I'm happy to change.

On Wed, Sep 30, 2015 at 12:58 PM, Jenny Schweers <notifications@github.com>
wrote:

> In corehq/apps/groups/templates/groups/all_groups.html
> <https://github.com/dimagi/commcare-hq/pull/8019#discussion_r40820114>:
>
> > @@ -35,6 +35,16 @@
> >  {% endblock %}
> >
> >  {% block main_column %}
> > +{% if needs_to_downgrade_locations %}
> > +    <div>
> > +        <p>
> > +            {% trans "This project no longer has access to the Organization feature." %}
> > +            <a href="{% url "downgrade_locations" domain %}">
> > +                {% trans "Click here to fix this." %}
>
> @dimagi/product <https://github.com/orgs/dimagi/teams/product> curious
> what the rationale was for that
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/8019/files#r40820114>.
>